docs/tutorials/kubectl/app-management-overview.md

```diff
@@ -0,0 +1,238 @@
+to mix-and-match styles for the same resource will result in undefined behavior.**
+
+| Management Style          | Environment             | Writers  | Operates Best On     | Learning Curve |
+|---------------------------|-------------------------|----------|----------------------|----------------|
+| Imperative Commands       | Dev or Small Clusters   | 1+       | Live Resources       | Lowest         |
+| Basic Yaml Config         | Production Clusters     | 1        | Individual Files     | Moderate       |
+| Yaml Config with Apply    | Production Clusters     | 1+       | Directories of Files | Highest        |
+
+# Imperative Commands
+
+## Command Examples
+
+Run an instance of the *nginx* container by creating a Deployment Resource
+
+```sh
+kubectl run nginx --image nginx
+```
+
+Same as above with different syntax (specifies resource type)
+
+```sh
+kubectl create deployment nginx --image nginx
+```
+
+## Command Overview
+
+*Kubectl* supports creating, updating, and deleting applications directly
+using the command line.  In this mode, the Kubernetes resources are
+configured through commandline flags.  This is the simplest way to get
+started or to run a one-off tasks in a cluster.  Because it does
+not provide an audit record or source of record outside what is live
+in the cluster, it may not be well suited for large production
+environments or critical applications.
+
+Advantages over *Config*
+
+- Commands are simple, easy to learn and easy to remember
+- Commands require only a single step to make changes to the cluster
+
+Disadvantages over *Config*
+
+- No review process for making changes
+- No audit trail associated with changes
+- No source of record beside what is live
+- No template for bootstrapping new applications
+
+<!---
+For a tutorial on how to use Imperative Commands for app management, see:
+[App Management Using Comands](/docs/tutorials/kubectl/app-management-using-commands/)
+-->
+
+# Basic Yaml Config
+
+## Config Example
+
+Create the resource defined in the configuration file.
+
+```sh
+kubectl create -f nginx_deployment.yaml
+```
+
+## Config Overview
+
+*Kubectl* supports creating, updating, and deleting applications by defining
+their configuration locally in a yaml or json config file, and using
+it as the basis to create, update or delete an application.
+
+This approach updates the application using `replace`, which will
+replace the existing resource spec in the cluster with the one provided
+to *kubectl*.
+
+Advantages over *Imperative Commands*
+
+- Config can be stored in a and SCM such as *git*
+- Config contains source of record for resource
+- Provides template for bootstrapping new applications
+
+Disadvantages over *Imperative Commands*
+
+- Config requires understanding resource definitions
+- Initial creation of resource requires additional step of creating the defintion in a file
+
+Advantages over *Yaml Config with `apply`*
```

Comment:
This advantage list doesn't hang together.  In fact, encouraging folks to use `replace` at all will often lead to a bad experience.  

TPR issues should go away at some point hopefully.  And "fewer edge cases" isn't descriptive enough for folks to really get any meaning.